src/handler/http/models/search.rs

```diff
@@ -0,0 +1,46 @@
+use config::meta::search as meta_search;
+use serde::{Deserialize, Serialize};
+use utoipa::ToSchema;
+
```

Comment:
Struct definitions here are all copy-pasted from `config::meta` since, prior to this PR, the JSON de/serialization logic in `config::meta` defines the HTTP API contracts.